membership_file/util.py

```diff
@@ -50,12 +50,13 @@ class MembershipRequiredMixin(LoginRequiredMixin):
         Mixin-equivalent of the @membership_required decorator.
     """
     fail_url = settings.MEMBERSHIP_FAIL_URL
+    requires_active_membership = True  # Boolean defining whether user should be active, or just linked as an (old) member
 
     def dispatch(self, request, *args, **kwargs):
         if request.member is None:
             # Current session has no member connected
             return HttpResponseRedirect(resolve_url(self.fail_url))
-        if not request.member.is_considered_member():
+        if not request.member.is_considered_member() and self.requires_active_membership:
```

Comment:
There's a mismatch between the default behaviour of this Mixin and the `@membership_required`-decorator.

Reply:
In  what way? The only thing that is different is that in the Mixin  one can disable the check for a member consideration. Which is by default taken into account just  like the decorator?

Reply:
Seems like I read some part of the code wrong! You're absolutely right!